Examples/Scripts/Python/ckf_tracks.py

```diff
@@ -1,12 +1,126 @@
 #!/usr/bin/env python3
 from pathlib import Path
-from typing import Optional
+from typing import Optional, Union
 
 from acts.examples import Sequencer, GenericDetector, RootParticleReader
 
 import acts
 
 from acts import UnitConstants as u
+from seeding import TruthSeedRanges
+
+
+@acts.examples.NamedTypeArgs(
+    truthSeedRanges=TruthSeedRanges,
+)
+def addCKFTracks(
+    s: acts.examples.Sequencer,
+    trackingGeometry: acts.TrackingGeometry,
+    field: acts.MagneticFieldProvider,
+    truthSeedRanges: TruthSeedRanges = TruthSeedRanges(),
+    outputDirCsv: Optional[Union[Path, str]] = None,
+    outputDirRoot: Optional[Union[Path, str]] = None,
+    selectedParticles: str = "truth_seeds_selected",
+) -> acts.examples.Sequencer:
+    """This function steers the seeding
+
+    Parameters
+    ----------
+    s: Sequencer
+        the sequencer module to which we add the Seeding steps (returned from addSeeding)
+    trackingGeometry : tracking geometry
+    field : magnetic field
+    outputDirCsv : Path|str, path, None
+        the output folder for the Csv output, None triggers no output
+    outputDirRoot : Path|str, path, None
+        the output folder for the Root output, None triggers no output
+    """
```

Comment:
- [ ] Missing `truthSeedRanges` description in docstring
- [ ] Missing `selectedParticles` description in docstring